bindings/c/test/apitester/tests/CApiBlobGranuleTenantCorrectnessMultiThr.toml

```diff
@@ -11,6 +11,7 @@ maxTenants = 5
 
 [[server]]
 blob_granules_enabled = true
+enable_encryption_at_rest = true
```

Comment:
I would suggest having two separate test profiles one with and another without encryption